src/point.class.ts

```diff
@@ -378,4 +386,6 @@ export class Point {
   }
 }
 
+const originZero = new Point(0, 0);
```

Comment:
i think is more readable defined as a const rather than in the function definition.
Does it get defined every function run if is in the function signature?

Reply:
good question
no idea